src/Servers/IIS/AspNetCoreModuleV2/CommonLib/fx_ver.cpp

```diff
@@ -7,26 +7,26 @@
 
 static bool validIdentifiers(const std::wstring& ids);
 
-size_t index_of_non_numeric(const std::wstring& str, size_t i)
+size_t index_of_non_numeric2(const std::wstring& str, size_t i)
```

Comment:
Why

Reply:
Ah because we copied this file and we get a name collision, I forgot about these, I'll try your namespace suggestion now to see if we can undo the renames

Reply:
Does that mean we can just use the runtimes version instead of rolling a copy?

Reply:
Probably, what does that look like in practice, defining our headers as externs or something? and just deleting the cpp file?

Reply:
Looks like I can just delete our fx_ver.cpp, and I just need to link our CommonLibTests.vxcproj against libnethost for this to work... one more time into mucking around with vcxproj!

Reply:
Added removing fx_ver to the meta issue https://github.com/dotnet/aspnetcore/issues/39281 to take up later, for some reason there's a linker error only in Win32/x86 without our implementation of fx_ver, so nuking fx_ver.cpp will be left to a future PR.  But at least we have a partially cleaned up CommonLibTests now for this PR